src/main/java/org/spongepowered/common/event/manager/InvokeEventListenerFactory.java

```diff
@@ -53,29 +51,29 @@ public AnnotatedEventListener create(Object handle, Method method) throws Except
 
     private static class InvokeEventHandler extends AnnotatedEventListener {
 
-        private final Method method;
+        private final ListenerClassVisitor.DiscoveredMethod method;
         private final EventFilter filter;
 
-        InvokeEventHandler(Object handle, Method method, EventFilter filter) {
+        InvokeEventHandler(final Object handle, final ListenerClassVisitor.DiscoveredMethod method, final EventFilter filter) {
             super(handle);
             this.method = checkNotNull(method, "method");
             this.filter = filter;
         }
 
         @Override
-        public void handle(Event event) throws Exception {
-            if (this.filter != null) {
-                Object[] filtered = this.filter.filter(event);
-                if (filtered != null) {
-                    StringBuilder args = new StringBuilder();
-                    for (Object o : filtered) {
-                        args.append(o.getClass().getName()).append(" ");
-                    }
-                    this.method.invoke(this.handle, filtered);
-                }
-            } else {
-                this.method.invoke(this.handle, event);
-            }
+        public void handle(final Event event) throws Exception {
+//            if (this.filter != null) {
```

Comment:
I don't think this can reasonably come back, maybe just delete this whole factory?

Reply:
Having reviewed it in SF 1.12, it doesn't even seem to have been used even then. Deleting!